admin/resources/js/components/classification/ClassificationTable.tsx

```diff
@@ -42,13 +44,18 @@ const messages = defineMessages({
     description:
       "Title displayed for the Classification table Maximum Salary column.",
   },
+  columnEditTitle: {
```

Comment:
Change:
`userTable.column.editTitle` to `classificationTable.column.editTitle`
and
_Title displayed for the User table Edit column_ to _Title displayed for the Classification table Edit column_